AppCenter/AppCenter/MSACAppCenter.m

```diff
@@ -171,6 +171,30 @@ + (void)setWrapperSdk:(MSACWrapperSdk *)wrapperSdk {
   [[MSACDeviceTracker sharedInstance] setWrapperSdk:wrapperSdk];
 }
 
++ (NSString *)countryCode {
+  return [[MSACDeviceTracker sharedInstance] getCountryCode];
+}
+
++ (MSACCustomProperties *)customProperties {
+  return nil;
```

Comment:
Returning `nil` here doesn't look like a good idea. We're not storing any property in the setter, we send it as a log instantly. So in this case I would rather keep it untouched, ie don't convert to a property. What do you think?